starknet_devnet/starknet_wrapper.py

```diff
@@ -547,6 +548,37 @@ async def load_messaging_contract_in_l1(
             self.starknet, network_url, contract_address, network_id
         )
 
+    async def postman_l1_to_l2(
+        self,
+        l1_contract_address: int,
+        l2_contract_address: int,
+        entry_point_selector: int,
+        payload: List[int],
+        nonce: int,
+    ) -> dict:
+        """Handles L1 to L2 message mock endpoint"""
+
+        state = self.get_state()
+
+        # Generate transactions
+        transaction = InternalL1Handler.create(
```

Comment:
Actually, why not use `Starknet.send_message_to_l2` directly?

Reply:
Calling it directly didn't reflect the blockchain state in our code base so I used `InternalL1Handler` last time and I was doing it analogically.